token/services/auditor/auditdb/db/badger/badger.go

```diff
@@ -39,11 +43,11 @@ type TransactionRecord struct {
 }
 
 type Persistence struct {
-	db *badger.DB
-
-	seq     *badger.Sequence
-	txn     *badger.Txn
-	txnLock sync.Mutex
+	db          *badger.DB
+	numGoStream int
```

Comment:
Plural? Steams?

Reply:
singular because it is number of go routines per stream 😅